.metadocrc

```diff
@@ -0,0 +1,28 @@
+{
+  "headerFile": "doc/header.md",
+  "footerFile": "doc/footer.md",
+  "minHeaderLevel": 2,
+  "removeInterface": true,
+  "files": [
+    "gs.js",
+    "lib/cursor.js",
+    "lib/provider.js",
+    "lib/fs.provider.js",
+    "lib/memory.provider.js",
+    "lib/pg.provider.js",
+    "lib/remote.provider.js"
+  ],
+  "customLinks": {
+    "pg.Pool": "https://github.com/brianc/node-pg-pool",
+    "pg.Client": "https://github.com/brianc/node-postgres",
+    "Uint64": "https://github.com/metarhia/common#class-uint64",
+    "Metaschema": "https://github.com/metarhia/metaschema/blob/master/lib/schema.js",
```

Comment:
If https://github.com/metarhia/metaschema/pull/231 is landed before this PR `Metaschema` link should be replaced with https://github.com/metarhia/metaschema#class-metaschema.